frontend_lib/README.md

```diff
@@ -1,32 +1,65 @@
+It includes all its dependencies, is self-contained, and can be imported using
+`require('tracim_frontend_lib')` or `import ... from 'tracim_frontend_lib'`.
 
-Problem: eslint will parse tracim_lib.js since it will come from another repository.
+Run:
 
-Solution: you must add `/* eslint-disable */` at the beginning and  `/* eslint-enable */` at the end of tracim_lib.js
+    yarn run build
 
-To automatize this: build tracim_lib with that command:
+This will produce two UMD modules:
+- `dist/tracim_frontend_lib.lib.js`: the library part
+- `dist/tracim_frontend_lib.style.js`: the styling part
 
-`npm run build && echo '/* eslint-disable */' | cat - dist/tracim_lib.js > temp && mv temp dist/tracim_lib.js && printf '\n/* eslint-enable */\n' >> dist/tracim_lib.js`
+The Webpack configuration used to build the standalone bundle is `webpack.config.js`.
 
-You can also create an alias for this in your .bashrc
+# Building as a browser library
+
+The browser library can be included using a `<script>` tag.
+It does not come with all its dependencies.
+Some of them are in the `tracim_frontend_vendors` bundle.
+The point of the browser library is to share the `tracim_fontend_lib` and common `dependencies` across the frontend, the apps and the lib in Tracim.
+
+The browser library can be used from a package (a Tracim frontend app) by accessing the global variable `tracim_frontend_lib` or in a package built with Webpack as long as:
+
+- `tracim_frontend_lib` is listed in the `externals` property of the Webpack configuration
+- all the libraries in `tracim_frontend_vendors` are also listed in `externals`
+- `tracim_frontend_vendors` and `tracim_frontend_libs` are included using the `<script>` tag before the app is loaded. In this configuration, `tracim_frontend_lib` expects the global object `tracim_frontend_vendors` to be available.
+
+This can be done by using the `externals.json` file built when bundling `tracim_frontend_vendors`.
+
+Run:
+
+    yarn run tracimbuild
+
+This will produce two browser libraries:
+- `dist/tracim_frontend_lib.tracim.lib.js`: the library part, declaring the `tracim_fontend_lib` global variable
+- `dist/tracim_frontend_lib.tracim.style.js`: the styling part
+
+The Webpack configuration used to build the browser library is `webpack.tracim.config.js`.
+
+# Frontend unit tests
+
+Frontend unit tests use the standalone bundle, hence the need to build it when building the full frontend.
+Using the browser libary does not work: `tracim_frontend_lib` needs to be compatible with CommonJS, and needs to access its dependencies. It is, however, impossible to make a `tracim_frontend_vendors` global object available for `tracim_frontend_lib` when using CommonJS modules, since their is no global namespace shared between the modules (in Node.js at least).
+
+# Building a Tracim app using `tracim_fontend_lib`
+
+In the code of your app, use `require('tracim_fontend_lib')` or `import ... from tracim_fontend_lib` to import the library.
```

Comment:
I would emphasize using the `import` keyword. And tell that you can also use `require` as a fallback

Reply:
```suggestion
In the code of your app, use `import ... from tracim_fontend_lib` to import the library (or  `require('tracim_fontend_lib')`).
```